specification/ai/DocumentIntelligence/models.tsp

```diff
@@ -335,33 +330,32 @@ union OperationStatus {
   @lroSucceeded
   succeeded: "succeeded",
 
-  @doc("The operation has completed.")
-  @lroSucceeded
-  completed: "completed",
-
   @doc("The operation has been canceled.")
   @lroCanceled
   canceled: "canceled",
+
+  @doc("The operation has been skipped.")
+  @lroCanceled
+  skipped: "skipped",
```

Comment:
Wondering whether this be a *terminal* state?

I think we'd generally discourage of adding a new terminal state in LRO status. User may not know that they need to handle this `skipped` state (unless they read the full doc).
Also note that it would affect SDK, as likely auto-generated code won't handle this `skipped` as the terminal of LRO.

Reply:
Skipped here is only returned for individual tasks in batch analysis result.  It is not part of any LRO status that will be consumed by SDK.  We could have created a different enum class just for individual batch tasks instead of reusing OperationStatus.  Feels like overkill though.

If clearer, we can remove the @lroCanceled decorator since it doesn't make any difference to the usage.

Reply:
I see. Yes, I am OK for a shared `OperationStatus`, as long as customer knows what to expect from the batch op and single op.